mne/time_frequency/tfr.py

```diff
@@ -245,6 +245,248 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
+        method == 'multitaper'. The number of good tapers (low-bias) is
+        chosen automatically based on this to equal floor(time_bandwidth - 1).
+    use_fft : bool, defaults to True
+        Use the FFT for convolutions or not.
+    decim : int | slice, defaults to 1
+        To reduce memory usage, decimation factor after time-frequency
+        decomposition.
+        If `int`, returns tfr[..., ::decim],
+        if `slice` returns tfr[..., decim].
+        .. note:
+            Decimation may create aliasing artifacts, yet decimation
+            is done after the convolutions.
+    output : str, defaults to 'complex'
+
+        * 'complex' : single trial complex.
+        * 'power' : single trial power.
+        * 'phase' : single trial phase.
+        * 'avg_power' : average of single trial power.
+        * 'itc' : inter-trial coherence.
+        * 'avg_power_itc' : average of single trial power and inter-trial
+          coherence across trials.
+
+    n_jobs : int, defaults to 1
+        The number of epochs to process at the same time. The parallization is
+        implemented across channels.
+    verbose : bool, str, int, or None, defaults to None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    out : array
+        Time frequency transform of epoch_data. If output is in ['complex',
+        'phase', 'power'], then shape of out is (n_epochs, n_chans, n_freqs,
+        n_times), else it is (n_chans, n_freqs, n_times). If output is
+        'avg_power_itc', the real values code for 'avg_power' and the
+        imaginary values code for the 'itc': out = avg_power + i * itc
+    """
+    # Check data
+    epoch_data = np.asarray(epoch_data)
+    if epoch_data.ndim != 3:
+        raise ValueError('epoch_data must be of shape '
+                         '(n_epochs, n_chans, n_times)')
+
+    # Check frequencies
+    if not isinstance(frequencies, (list, np.ndarray)):
+        raise ValueError('frequencies must be an array-like.')
+    frequencies = np.asarray(frequencies, dtype=float)
+    if frequencies.ndim != 1:
+        raise ValueError('frequencies must be of shape n_freqs.')
+
+    # Check sfreq
+    if not isinstance(sfreq, (float, int)):
+        raise ValueError('sfreq must be a float or an int.')
+    sfreq = float(sfreq)
+
+    # Check output
+    allowed_ouput = ('complex', 'power', 'phase',
+                     'avg_power_itc', 'avg_power', 'itc')
+    if output not in allowed_ouput:
+        raise ValueError("Unknown output type. Allowed are %s but "
+                         "got %s" % (allowed_ouput, output))
+
+    if method not in ('multitaper', 'morlet'):
+        raise ValueError('method must be "morlet" or "multitaper"')
+
+    # Default zero_mean = True if multitaper else False
+    zero_mean = method == 'multitaper' if zero_mean is None else zero_mean
+    if not isinstance(zero_mean, bool):
+        raise ValueError('')
```

Comment:
what's the situation where this would happen? Doesn't the `method == 'multitaper'` force `zero_mean` to be bool?